<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialogClass</class>
 <widget class="QDialog" name="SettingDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>82</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>295</width>
    <height>82</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>295</width>
    <height>82</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Setting Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sort Order</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbSortOrder">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>InspectID(Greater)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>InspectID(Lesser)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>InspectTime(Greater)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>InspectTime(Lesser)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NCCount(Greater)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NCCount(Lesser)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NCCountOnlyFront(Greater)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NCCountOnlyFront(Lesser)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NCCountOnlyBack(Greater)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NCCountOnlyBack(Lesser)</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>168</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
     <property name="icon">
      <iconset resource="ShowHistoryListForReview.qrc">
       <normaloff>:/Qtres/flash_swf_64.PNG</normaloff>:/Qtres/flash_swf_64.PNG</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbSortOrder</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources>
  <include location="ShowHistoryListForReview.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
